_includes/install_command.html

```diff
@@ -0,0 +1,49 @@
+{%- capture code -%}
+brew {{include.modifier}} install {{include.item}}
+{%- endcapture -%}
+<style>
+    .inline {
+        display: inline-flex;
+        background: rgba(0, 0, 0, 0.3);
+        border-radius: 0.4em;
+        padding-right: 8px;
+    }
+
+    .inline .highlight {
+        margin: 0;
+    }
+    
+    .inline .highlight pre {
+        margin: 0;
+    }
+    
+    #copyButton {
+        background: transparent;
+        border: none;
+        border-left: 1px solid #aaa;
+    }
+</style>
+
+Install command:
+<div class="inline">
+    {% highlight bash %}
+    $ {{- code -}}
+    {% endhighlight %}
+
+    <button
+        id="copyButton"
+        aria-label="Copy to clipboard"
+        style="visibility: hidden"
```

Comment:
Feels like this should be defined in the CSS?

Reply:
If it is in CSS like it was before, then `document.getElementById("copyButton").style.removeProperty("visibility");` won't work, as it is modifying the style on the actual element.

Reply:
What about using `display: none` instead?